### DISCUSSION TITLE
Looking to Contribute GUI or Some Geometric Algorithms

### DISCUSSION THREAD

**Discussion by nadavsuissa** [pygae/galgebra#501] [Ideas]
Hello, We are a group of Vs students looking to contribute to an open source code, we would like to implement a Graphic interface or some interesting geometric algorithms and are questions are the following
Is someone already working on this?
I not is this something that you might be interest in, If we implement this to a high enough level, would you accept the contribution?

**Comment by eric-wieser** [comment]
Hi! It's great to see some interest in galgebra.

It's not clear to me exactly what you're proposing here; what do you want the graphic interface to do, and what geometric algorithms are you thinking of?
